components/bunnygrabber.py

```diff
@@ -1,19 +1,31 @@
+# import Robot py library so this file can interface with the robot.
 import wpilib
 
-
+# Class for controlling arm, will be instantiated in later script usage.
 class BunnyGrabber:
-    """Controls the bunny grabber.
-    """
+#	This program controls the bunny grabber arm.
+
+#   Attach variable to motor arm so we can tell the motor which direction to move.
     bunny_motor = wpilib.Talon
+#   Declare direction variable so we can change which direction the motor moves in.
     direction = 0
 
+
+#	The following methods set_backward() and set_forward() tells the motor that
+#	when it moves via the execute() method which direction it should move in.
+
+
     def set_backward(self):
         self.direction = -1
 
     def set_forward(self):
         self.direction = 1
 
+#   Method which allows the motor to move (or stop) based on if a direction has been specified with the previous methods.
     def execute(self):
+
```

Comment:
This should be condensed a bit, into something like "Sets motor direction based off the `direction` variable"